packages/components/src/templates/header.stories.ts

```diff
@@ -0,0 +1,78 @@
+import '../solid-components';
+import { html } from 'lit-html';
+
+export default {
+  tags: ['!dev'],
+  title: 'Templates/Header'
+};
+/**
+ * Sample of the sd-header
+ *
+ * ```
+ * ```
+ */
+export const Default = {
+  render: () => html`
```

Comment:
How it is presented in the current story is better:
<img width="861" alt="Bildschirmfoto 2024-09-06 um 17 36 20" src="https://github.com/user-attachments/assets/1a8bfdd8-6930-40d6-adbf-c84b0a7f19b1">

ATM the available width is sth like 980px which should look like these here. Responsive behavior not working properly?
![Bildschirmfoto 2024-09-06 um 17 38 32](https://github.com/user-attachments/assets/1d876b45-3d31-48b2-8d0a-839688607e66)

Reply:
Created second template (Sample B) as discussed in sync.

Regarding the responsive behaviour question: the sidebar on the left and the TOC on the right interfer with the responsive behaviour. If you close the sidebar the breakpoints will work more closely to what is expected.